satpy/readers/seviri_l1b_native.py

```diff
@@ -329,7 +329,7 @@ def get_area_extent(self, dataset_id):
 
         if dataset_id.name == 'HRV':
             grid_origin = data15hd['ImageDescription']['ReferenceGridHRV']['GridOrigin']
-            center_point = HRV_NUM_COLUMNS / 2
+            center_point = (HRV_NUM_COLUMNS-2) / 2
```

Comment:
with 11136 columns the `center_point` with this formula is 5567. according to the documentation the center point should be 5566. should this be `center_point = (HRV_NUM_COLUMNS / 2) - 2`? or am i missing something here?

Reply:
You are right, thanks for spotting this. Sorry, dumb mistake. I wonder when area extents will stop confusing me....